project_1/src/main/webapp/client/views/login.jsp

```diff
@@ -57,8 +57,9 @@
                             <input type="submit" value="Login"
                                    style="margin-top: 15px; background-color: #FF742D; color: white">
                         </div>
-                        <a href="#"><span style="text-indent: 20px"><spring:message code="label.lost.password"/> </span></a>
+                        <a href="/register"><span style="text-indent: 20px">Register</span></a>
```

Comment:
` style="text-indent: 20px">` (yaoming) toàn bộ style bên trong tag em đều khai báo css hết nhé, cái này bị rất nhiều, em tạo 1 pull để fix đi

Reply:
Cai này e fix sau chị ơi.